Automattic/wp-calypso
# Posts List: add "Recent views" to post glance

This adds "Recent views" to the `PostActionCount` in the Posts list. The original (total) views was removed in #26711 due to performance issues for older posts. The new stat shows a post's views in the last 30 days.

![recent-views](https://user-images.githubusercontent.com/942359/45451799-2b2f0f00-b6aa-11e8-870b-1fb4ec6a0bef.png)

Fixes #26894.

**To test:**
- Apply D17740-code and sandbox `public-api.wordpress.com`.
- Visit a site's posts list.
- For posts with views in the past 30 days, a "{count} Recent Views" should be shown, with a link title of "{count} recent views in the past 30 days". This should link to the full stats view for that post.
- For posts without views in the past 30 days, no recent views should be shown.
